templates/GuardDuty/trusted-ips-bucket.yaml

```diff
@@ -0,0 +1,53 @@
+# From https://github.com/org-formation/org-formation-reference/tree/master/src/templates/070-guard-duty
+AWSTemplateFormatVersion: '2010-09-09'
+
+Metadata:
+  cfn-lint:
+    config:
+      ignore_checks: [W2001]
+
+Parameters:
+  resourcePrefix:
+    Type: String
+
+  bucketName:
+    Type: String
+
+Resources:
+  TrustedIpsBucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain
+    UpdateReplacePolicy: Retain
+    Properties:
+      BucketName: !Ref bucketName
+      AccessControl: Private
+      PublicAccessBlockConfiguration:
+        BlockPublicAcls: true
+        BlockPublicPolicy: true
+        IgnorePublicAcls: true
+        RestrictPublicBuckets: true
+      BucketEncryption:
+        ServerSideEncryptionConfiguration:
+          - ServerSideEncryptionByDefault:
+              SSEAlgorithm: AES256
+
+  TrustedIpsBucketPolicy:
+    Type: AWS::S3::BucketPolicy
+    Properties:
+      Bucket: !Ref TrustedIpsBucket
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Sid: GuardDutyRead
+            Effect: Allow
+            Principal:
+              Service: guardduty.amazonaws.com
+            Action:
+              - s3:GetObject
+            Resource: !Sub '${TrustedIpsBucket.Arn}/trusted_ips.txt'
```

Comment:
Maybe it doesn't matter if the trusted IPs list is empty... but do want to have a list like this?

Reply:
do you have a concern with it being here?  if not then lets just add as an optional feature.  if you have a concern later you are free to remove it if you want.